roles/rook/README.rst

```diff
@@ -0,0 +1,115 @@
+Rook
+====
+
+This role is used for updating and recovering the rook jenkins in the rook ci vpc.
+
+Those are the functions in this role are
+
+- rook-jenkins-update For updating rook jenkins version to the version defined in the "jenkins_master_image" varible
+
+- rook-os-update For updating rook jenkins OS packages
+
+- rook-recovery For recoverying the Prod-jenkins instance from the image defined in the "image" varible in a case that the instance was deleted or crashed
+
+Usage
++++++
+
+The rook role is primarily used by the ``rook.yml`` playbook.  Run this playbook with one of the optional tags listed in the ``rook.yml``
+to upgrade OS packages/recover from image or update the jenkins app on the rook jenkins.
+
+**NOTE:** ``rook.yml`` Is currently using only localhost and not any host from the inventory this is because the rook-recovery play deploys and configures the rook jenkins during his run.
+
+**NOTE:** Before running ``rook.yml`` make sure your ip address has ssh access to the vpc this is configured from the aws dashboard, "rook-jenkins-group" security group inbound roles.
+
+- Deploying rook jenkins from image in case of a crash/corruption
```

Comment:
Take a look at how this is getting rendered: https://github.com/ceph/ceph-cm-ansible/blob/31323d80631a20f096d2270b2e1a5b8ab4b2026a/roles/rook/README.rst#usage

I'd like for it to be cleaner.  Here's an online tool I've found helpful to help write rst: http://rst.ninjs.org/#